src/main/java/org/apache/sling/auth/oauth_client/impl/OidcAuthenticationHandler.java

```diff
@@ -120,7 +126,7 @@ public class OidcAuthenticationHandler extends DefaultAuthenticationFeedbackHand
         @AttributeDefinition(name = "Path",
                 description = "Repository path for which this authentication handler should be used by Sling. If this is " +
                         "empty, the authentication handler will be disabled. By default this is set to \"/\".")
-        String path() default "/";
+        String[] path() default {"/"};
```

Comment:
Should we rename to paths?

Reply:
Actually it is from: https://github.com/apache/sling-org-apache-sling-auth-core/blob/47eee11b0a05bdd3d20fcbfba81b2077f503f058/src/main/java/org/apache/sling/auth/core/spi/AuthenticationHandler.java#L44-L57

Reply:
Make sense, thanks.